test/onnx/test_fx_to_onnx.py

```diff
@@ -68,9 +68,6 @@ def func(x):
             func, torch.randn(1, 1, 2), export_options=self.export_options
         )
 
-    @unittest.skip(
-        "max_pool2d is not supported in ATen Lib: https://github.com/microsoft/onnx-script/issues/585"
-    )
     def test_mnist(self):
```

Comment:
If this test finally alives, could you also update it back to runtime.py?

Reply:
Sorry what should I change?

Reply:
Find test_mnist in test_fx_to_onnx_with_runtime.py

Reply:
Looks like that was a flaky test not related to the missing op?

Reply:
Yes, but these two were originally created together, but we didn't have maxpool2D, so we changed the one in runtime, and skip this one. I just thought it would be better if we changed it back.

Reply:
SG! Let me try that

Reply:
Done